### PR TITLE
Add news items from PR.

### PR DESCRIPTION
**Pull request by teh**

### CODE REVIEW

**Comment by johbo** [comment]
Yeah! Thanks for helping out 👍

File: content/2017/04-everything-not-saved-will-be-lost.rst [hunk 1]
```diff
@@ -4,12 +4,25 @@
...
 
 News
 ====
 
-TODO:
+- We now allow packages to be marked insecure, and you have to opt-in to install insecure packages similarly to how you opt in to non-free software.  The option is called ``permittedInsecurePackages``. See the `updated docs`_ for more.
+
+- Robin finished the `systemd 232 update`_ which opens up several new `opportunities for hardening`_. It will also allow us to move away from hard-coded user IDs for less important services. Just use ``DynamicUser = true``.
+
+- Thanks to Graham we now have `aarch64 support`_, and it's even being build Hydra! Many of you will have noticed already from the beatiful stream of emails for each succeeding and failing package.
```
Comment (globin) [inline content/2017/04-everything-not-saved-will-be-lost.rst:19]: s/build/built

File: content/2017/04-everything-not-saved-will-be-lost.rst [hunk 2]
```diff
@@ -4,12 +4,25 @@
...
+.. _`17.03 milestone`: https://github.com/NixOS/nixpkgs/milestone/10
+
 
 News
 ====
 
-TODO:
+- We now allow packages to be marked insecure, and you have to opt-in to install insecure packages similarly to how you opt in to non-free software.  The option is called ``permittedInsecurePackages``. See the `updated docs`_ for more.
+
+- Robin finished the `systemd 232 update`_ which opens up several new `opportunities for hardening`_. It will also allow us to move away from hard-coded user IDs for less important services. Just use ``DynamicUser = true``.
```
Comment (globin) [inline content/2017/04-everything-not-saved-will-be-lost.rst:17]: This actually happened in January, don't know if you want to include this, I don't mind :)

Comment (johbo) [reply content/2017/04-everything-not-saved-will-be-lost.rst:17]: @globin dunno if we mentioned it already, if not we should keep it IMO (at least on my radar it did not yet appear :D )

**Pull request merged**